frontend/jsconfig.json

```diff
@@ -8,5 +8,11 @@
       "@@/*": ["./*"]
     }
   },
-  "exclude": ["node_modules", ".nuxt", "dist"]
+  "exclude": ["node_modules", ".nuxt", "dist"],
+  "include": [
+      "**/*.js",
+      "**/*.vue",
+      ".eslintrc.js",
+      ".eslintrc.typescript.js"
```

Comment:
Why do you need to include it into jsconfig?

Reply:
Without it this error gets thrown:

```bash
The file does not match your project config: .eslintrc.js.
The file must be included in at least one of the projects provided. eslint
````

So the files get found by eslint but it doesn't know what to do with them. There's then 2 options to fix this:

1. Ignore files - add to .eslintignore
2. Manually tell to lint them - what I did

I thought manually adding & linting them is probably better than just ignoring them, right?

Reply:
> I thought manually adding & linting them is probably better than just ignoring them, right?

Correct! Thanks for the explanation